doc/contrib/pr.rst

```diff
@@ -48,6 +65,54 @@ Refer to the `Github
 documentation <https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request>`__
 for more explanation of the Git workflow described above.
 
+.. _contribpr-collab:
+
+Collaborating on a PR
+---------------------
+
+Especially for larger PRs, it may be a good idea to collaborate with others. There are various ways to do so,
+for instance to have a shared branch somewhere in a fork of the Arbor repo. Since we're using Github, you can
+also add commits to a PR opened by someone else. Since the correct procedure for that is 
+`Github specific <https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork>`_,
+here the steps for how to do this (with just ``git``).
+
+Alongside the instruction, an example situation will be given assuming you have the Arbor main
+repo setup as a remote called ``upstream``, you want to commit to pull request number ``321``, opened by a 
+Github user called ``github_user_007``, who named the branch they're trying to merge ``my_special_branch``,
+in their fork of the Arbor repo called ``arbor-sim`` (which is the default when you fork on Github).
+
+Situation as a table
+~~~~~~~~~~~~~~~~~~~~
+
+=============================== ========================= ======================
+description                     variable                  example
+remote for Arbor main repo      ``$REMOTE``               ``upstream``
+PR you want to contribute to    ``#PR``                   ``321``
```

Comment:
You could make this `$PR`, then people could actually for real use shell vars. Otherwise the whole `$` is just a gimmick. Also add a quick note like

> These are written as (bash) shell variable so you could follow the session by
```sh
export REMOTE=upstream
# or REMOTE=upstream, but caps call for export
```